tests/security/test_blobstorage.py

```diff
@@ -17,7 +17,7 @@ def test_blobstorage(config, sample_input, encrypt):
 def test_encoded_format(config, sample_input, encrypt):
     config["ENCRYPT_SECRETS"] = encrypt
     encoded = encode_blob(sample_input)
-    assert encoded.startswith(chr(encrypt) + "\x00\x00\x00\x00")
+    assert encoded.startswith((b"\x01" if encrypt else b"\x00") + b"\x00\x00\x00\x00")
```

Comment:
We want to work with bytes here both in Python 2 and Python 3 since this is binary data.